<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="hostText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="portSpin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QPushButton" name="linkBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>link</string>
   </property>
  </widget>
  <widget class="QPushButton" name="canBtn">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>750</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>discon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>241</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userpassword">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>241</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>671</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>230</y>
     <width>271</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>163</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>163</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>150</y>
     <width>163</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>750</y>
     <width>671</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>750</y>
     <width>163</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>canBtn</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
